<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SourceWidget</class>
 <widget class="QWidget" name="SourceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>41</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QPushButton" name="pushButton_chooseMovie">
     <property name="text">
      <string>Choose Movie</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_fps">
     <property name="text">
      <string>FPS:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="spinBox_fps"/>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="format">
      <string>%v|%m</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
